doc/source/whatsnew/v0.19.0.rst

```diff
@@ -731,8 +731,8 @@ A ``Series`` will now correctly promote its dtype for assignment with incompat v
 
 .. _whatsnew_0190.api.to_datetime_coerce:
 
-``.to_datetime()`` changes
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Method ``.to_datetime()`` changes
```

Comment:
@datapythonista should this be Function instead of Method?

Reply:
Yes, I think so.